<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_connect">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect guncon</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_guncon">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calibration">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Calibration</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_run">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Run</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>39</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixmap_mirilla">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>171</y>
      <width>101</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="img_audio.qrc">:/img/mirilla.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_absx">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>39</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_absy">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>260</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_absy_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>39</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_absx_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>110</y>
      <width>65</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>4:3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkbox_mouse">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>130</y>
      <width>65</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>mouse</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_guncon2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>150</y>
      <width>65</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>guncon2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_ps2_cal">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>150</y>
      <width>81</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>ps2 calibration</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_serial">
    <property name="geometry">
     <rect>
      <x>127</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_serial_connect">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>80</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Serial Connect</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="txt_log">
    <property name="geometry">
     <rect>
      <x>173</x>
      <y>300</y>
      <width>211</width>
      <height>70</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_conf">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>conf</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="trigger_r_button">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>74</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="img_audio.qrc"/>
 </resources>
 <connections/>
</ui>
